src/main/scala/com/databricks/spark/redshift/RedshiftRelation.scala

```diff
@@ -107,7 +108,7 @@ case class RedshiftRelation(jdbcWrapper: JDBCWrapper, params: MergedParameters,
     val filterClauses = filters map {
       case EqualTo(attr, value) => s"${sqlQuote(attr)} = ${compileValue(value)}"
       case LessThan(attr, value) => s"${sqlQuote(attr)} < ${compileValue(value)}"
-      case GreaterThan(attr, value) => s"${sqlQuote(attr)}) > ${compileValue(value)}"
```

Comment:
Yeah, there's a separate earlier PR just for that one line... got pulled into this branch by accident as I'm trying to get things working